styles.css

```diff
@@ -1,8 +1,16 @@
-.wrapper {
+.body {
+  padding: 0;
+  margin: 0;
+  background-color: #e5e5e5;
+}
```

Comment:
- Please remove the extra space surrounding your design. You can easily fix this by changing `.body` to `body`. Using the dot selector here doesn't make sense, since the `body` is an element and not a class.

Reply:
@gemmen29.
 Kindly see changes made as was indicated to help me approve before re-submitting for code review.

Reply:
@gemmen29 
Kindly see changes made as was indicated to help me approve before re-submitting for code review

Reply:
You did all changes correctly except for the background image of the contact form you use the same background you used in the `hero` section.